site/app/templates/admin/Configuration.twig

```diff
@@ -278,6 +281,17 @@
             }
         });
 
+        $(document).on('change', '#forum_enabled', function(){
+            var forum_enabled = $('#forum_enabled').is(":checked");
```

Comment:
There is a slight issue with the display. If I refresh the page (where #forum_enabled) was not changed the check is off but the text still displays. 

![uncheckedStillDisplayed](https://user-images.githubusercontent.com/16356240/59581800-07b74b80-90a4-11e9-8296-0222299814ae.png)